doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -320,7 +320,7 @@ Performing computations
    ref_or_pattern_occ ::= @reference {? at @occs_nums }
    | @one_term {? at @occs_nums }
    occs_nums ::= {+ {| @natural | @ident } }
-   | - {| @natural | @ident } {* @int_or_var }
+   | - {+ {| @natural | @ident } }
```

Comment:
Didn't you intended to write `nat_or_var` here and in `orderedGrammar`, as it is in `fullGrammar`?

Reply:
For now, this nonterminal is "spliced". I am in favor of not splicing it anymore, but Jim said that we should address this question separately as part as a documentation PR like #13343.

Reply:
I already changed this in #13343.  This change was automatically generated by doc_grammar based on `fullGrammar` and the editing instructions in `common.edit_mlg` , BTW.